Yoast/wordpress-seo
# 11516 add recalibration beta notice

## Summary

This PR can be summarized in the following changelog entry:

* Shows a notice when recalibration beta has been enabled yet.

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch
* Make sure you haven't enabled the recalibration beta under features
* Go to the dashboard
* See the notice and dismiss it
* Refresh the page
* Notice is gone!
* In the database remove the usermeta with the name `wpseo-calibration-meta-notification`
* Reload the dashboard and see the notice is back
* Enable the recalibration beta under features
* The notice should be gone!

## Quality assurance


Fixes #11516 
Requires #11570 -> merge #11570 first!
